xonsh/prompt/gitstatus.py

```diff
@@ -1,13 +1,21 @@
 # -*- coding: utf-8 -*-
 """Informative git status prompt formatter"""
 
-import os
 import builtins
+import collections
+import os
 import subprocess
 
 import xonsh.lazyasd as xl
 
 
+@xl.lazyobject
+def GitStatus():
```

Comment:
But named tuples are just type definitions, so they probably shouldn't be lazily implemented.  We don't hide class definitions in lazy objects, for example.  I think this is overdoing it.